specification/subscription/resource-manager/readme.md

```diff
@@ -23,6 +23,13 @@ To see additional help and options, run:
 ``` yaml
 directive:
   - suppress: R2059
+  - suppress: PutResponseCodes
+    from: subscriptions.json
+    where: $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/changeTenantRequest/default"].put
+    reason: Not supposed to return 201 as the response code for the below API since existing api with new version change, got exceptions from ARM reviewer.
```

Comment:
I'm not very clear on the reason here. Is this API introduced in this PR? I didn't find this API in the `2019-10-01-preview` version.

Reply:
@gary-x-li , it's actually not present in our public swagger, but we are using the api in 2019-10-01-preview , example - https://msazure.visualstudio.com/One/_git/AD-CAB-Provisioning?path=/src/WebApps/SubscriptionRP/Controllers/V2019_10_01/TargetDirectoryController.cs&version=GBdev&line=58&lineEnd=61&lineStartColumn=9&lineEndColumn=67&lineStyle=plain&_a=contents, 
Didn't know why the swagger is not present in an old preview version, probably missed adding it or there are some different process that time.